src/Nethermind/Nethermind.DataMarketplace.Test/ContractInteractionTest.cs

```diff
@@ -380,7 +372,7 @@ public FilterLog[] GetFilterLogs(int filterId)
                 throw new NotImplementedException();
             }
 
-            public FilterLog[] GetLogs(BlockParameter fromBlock, BlockParameter toBlock, object address = null, IEnumerable<object> topics = null)
+            public IEnumerable<FilterLog> GetLogs(BlockParameter fromBlock, BlockParameter toBlock, object address, IEnumerable<object> topics)
```

Comment:
do you actually manage to serialize it directly to the body stream?

Reply:
This is jusst a test mock, but for real class yesit enumerates and serializes.